src/Compilers/Test/Utilities/CSharp/CSharpTestBase.cs

```diff
@@ -2330,17 +2330,6 @@ protected static CSharpCompilation CreateCompilationWithIndexAndRangeAndSpan(CSh
                 options: options,
                 parseOptions: parseOptions);
         }
-
-        protected static CSharpCompilation CreateCompilationWithSpanAndMemoryExtensions(CSharpTestSource text, CSharpCompilationOptions options = null, CSharpParseOptions parseOptions = null)
```

Comment:
Do we consider this low risk of hitting merge conflicts? I suppose no other features are being actively developed that would possibly use this helper.